index.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+const fsCallback = require("./lib/files-callBack.js");
+const fsPromise = require("./lib/file-promise.js"), write = require("./lib/write");
+const file = process.argv.slice(2)[0];
+
+
+
+/** this is just a function to run 
+ * callBack with two parameters */
+
+function myTestcallBack(err, data) {
+  console.log("hi!")
+}
+/**use callBack read write read
+ * read l
+ * @param file
+ * @param callBack
+ * write
+ * change data field value 
+ * @param file
+ * @param data
+ * @param callBack
+ * read ll
+ * @param file
+ * @param callBack
+ */
+const useCallbacks = cb => {
+  fsCallback.read(file, (err, data) => {
+    if (err) {
+      console.error(err);
+    } else
+      data.lastName = "Callback";
+    fsCallback.write(file, data, (err, result) => {
+      if (err) {
+        console.error(err);
+      } else {
+        fsCallback.read(file, (err, afterData) => {
+          cb(afterData)
+        })
+      }
+
+    })
+  })
+}
+
+/** 
+ * use Promise
+ *  read l
+ * @param file
+ * change data field value
+ * @return data
+ * write
+ * @param file
+ * @param obj then 
+ * read ll
+ * @param file
+ */
```

Comment:
Your JSDoc comments should be about the current function being defined, not all the functions called within the current function. So, in this example, the JSDocs comments should just tell me what usePromise does, what parameters usePromise requires, and what usePromise returns.

https://devhints.io/jsdoc